examples/app-with-scenes/src/pages/WithDrilldown/WithDrilldown.tsx

```diff
@@ -70,7 +70,7 @@ const getDrilldownsAppScene = () => {
 
               return new SceneAppPage({
                 url: prefixRoute(`${ROUTES.WithDrilldown}`) + `/room/${roomName}/temperature`,
-                title: `${roomName} overview`,
+                title: `${decodeURI(roomName)} overview`,
```

Comment:
This change is fixing an issue that was addressed in old dead repo (see [here for context](https://github.com/grafana/scenes-app-template/pull/2))